README.md

```diff
@@ -96,6 +96,22 @@ Yes, it's really all you need to get started as you can see in this live and int
 
 [![Edit Button](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/4j7m47vlm4)
 
+## Should I start with v1-beta?
+
+Yes, you should. We often get this question:
+
+> Should I start with v1-beta? But beta is a beta, so it's not a final product and I'm not guaranteed anything.
+
+People are starting projects with v0.x which given the quality and stability of v1 they shouldn't be. The v1-beta effort started in May 2016.
+There are lots of us using v1-beta in production already with no problems and the odd breaking change is less hassle than upgrading from v0.x to v1-beta.
+They are just creating extra work for themselves as they will have to transition at some point.
+
+Material-UI will never be a final product, you will never be guaranteed anything neither with v0.x nor v1, v2, etc.
```

Comment:
"whether with v0.x, v1, or an future release."

Reply:
Really?! Wow, I didn't know `an future` was English.

Reply:
That's a new one for me as well.

Reply:
Bah, "any", sorry. Typos in suggested changes are not good! 😦

Reply:
Ohhh

Reply:
@mbrookes Do you mind directly committing on v1-beta? :)

Reply:
Will do.